internal/cmd/launcher.go

```diff
@@ -194,7 +194,7 @@ func k6buildProvision(gs *state.GlobalState, deps k6deps.Dependencies) (commandE
 	}
 
 	if config.BuildServiceAuth == "" {
-		return nil, errors.New("k6 cloud token is required when Binary provisioning feature is enabled." +
+		return nil, errors.New("k6 cloud token is required when the binary provisioning feature is enabled." +
```

Comment:
```suggestion
		return nil, errors.New("k6 cloud token is required when the binary provisioning feature is enabled." +
```
I'm using Binary provisioning because it is the _name_ of the feature. So, any name requires to be capitalized.

Reply:
I think it should be "Binary Provisioning"

Reply:
It is fine to me, go for it :+1: